app/models/access_token.rb

```diff
@@ -138,7 +138,10 @@ def validate_scope_exists
   end
 
   def expires_at=(value)
+    return if value.blank?
+
     DateTime.strptime(value)
+
     super value
```

Comment:
hmm, excuse my ignorance, what should `super` be here? Do we `#strptime` jut to catch exceptions?

Reply:
ok, got it, this is an attribute. So this is sort of compensating for broken values.